### PR TITLE
Update to v1.8.5

### PR DESCRIPTION
**Pull request by legowerewolf**
Also includes script for automatically updating based on latest available version on https://pkgs.tailscale.net/stable/

### CODE REVIEW

File: tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -4,7 +4,7 @@ $toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
 Get-ChildItem -Path "$toolsDir\tailscale.cer" | Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher
 
-$url = 'https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.8.3.exe'
+$url = 'https://pkgs.tailscale.com/stable/tailscale-ipn-setup-1.8.4.exe'
```
Comment (bradfitz) [inline tools/chocolateyinstall.ps1:7]: 1.8.5 is out now, sorry.

File: updater.ps1 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+$channel = "https://pkgs.tailscale.com/stable/"
+
+$url = $channel + ((Invoke-WebRequest $channel).links | where-object { $_.href -like "*.exe" }).href
```
Comment (bradfitz) [inline updater.ps1:3]: This seems fragile. How about you use https://pkgs.tailscale.com/stable/?mode=json&os=windows instead?

Comment (legowerewolf) [reply updater.ps1:3]: I knew there was a JSON mode, but I couldn't find any documentation on it.

File: tools/chocolateyinstall.ps1 (hunk 1 above)
Comment (legowerewolf) [reply tools/chocolateyinstall.ps1]: Bumped to 1.8.5.

File: updater.ps1 (hunk 2 above)
Comment (legowerewolf) [reply updater.ps1]: Updated.

**Pull request merged**